tests/SprykerEcoTest/Zed/PunchoutCatalogs/Business/PunchoutCatalogsFacadeTest.php

```diff
@@ -27,6 +27,12 @@ class PunchoutCatalogsFacadeTest extends Unit
     protected const CONNECTION_USERNAME = 'Test username';
     protected const CONNECTION_TYPE = 'Test type';
     protected const CONNECTION_FORMAT = 'Test format';
+    protected const CONNECTION_PASSWORD = 'Test password';
+
+    /**
+     * @see \SprykerEco\Zed\PunchoutCatalogs\Business\Writer\PunchoutCatalogsWriter::VAULT_DATA_TYPE_PASSWORD
```

Comment:
`@uses`?

`@see` as for me is better to be used to show usage places (such as controller actions, etc).